### DISCUSSION TITLE
Theme browser and downloader for the MM Plus

### DISCUSSION THREAD

**Discussion by SebastianMaung** [OnionUI/Onion#1287] [Show and tell]
Hey everyone, today I just completed my [theme browser and downloader ](https://github.com/SebastianMaung/MiyooThemeDownloader)for the Miyoo Mini Plus, so you can browse and download themes straight from your Miyoo. It grabs the themes from the Onion repo of themes, and uses the Advance Menu to browse the previews before you download them. Big thanks to the people who maintain the [theme repo](https://github.com/OnionUI/Themes), it's well maintained and was very easy for me to make this because of them. Would be really cool if someone added this app to the new Onion OS site.

**Comment by FDrebin** [comment]
Did you create a pull request to add this app yet to stock Onion?

**Reply by SebastianMaung** [reply]
I was thinking that it could be added to the [community apps section](https://onionui.github.io/docs/community-apps) unless its a better idea to add it to the stock os, not sure if the app is polished enough

**Reply by FDrebin** [reply]
I'd submit a pull request for your app, include as much documentation as you can and see what happens, maybe you'll get help polishing it if it isn't "pretty enough".

**Reply by lachlanmaclean** [reply]
This would make a great addition to stock Onion, definitely submit a PR and good luck :)
